cmd/main.go

```
@@ -0,0 +1,33 @@
+	token := os.Getenv("NETBOX_TOKEN")
+
+	c, err := client.New(url, token)
+	if err != nil {
+		fmt.Println(err.Error())
+		os.Exit(1)
+	}
+	h := handlers.New(c)
+
+	http.HandleFunc("/api/devices/", h.Device)
+	fmt.Println("Listening on port 8080")
+	http.ListenAndServe(":8080", nil)
```

golangci-lint (1.64.8)

Error return value of `http.ListenAndServe` is not checked

(errcheck)